desktop/assets/texts/es-ES/preferences_dialog.ftl

```diff
@@ -28,5 +28,5 @@ gamemode = GameMode
 gamemode-tooltip =
     GameMode aplica temporalmente un set de optimizaciones a tu computadora y/o a Ruffle.
     Ruffle solicita GameMode solo cuando se está reproduciendo una película.
-gamemode-default = Por defecto
-gamemode-default-tooltip = GameMode solo será habilitado cuando la preferencia de poder esté en alto.
+gamemode-default = Predeterminado
```

Comment:
Are you sure that this is better, @kjarosh?

Reply:
I'm not sure, it's not _obviously_ wrong for me, so I trusted the translator